docs/manuals/gui/viselements/charts/funnel.md

```diff
@@ -0,0 +1,359 @@
+| [*options*](../chart.md#p-options)  | dictionary  | `connector` can be used to customize the graphical properties of the lines connecting the boxes. |
+
+## Examples
+
+### Simple funnel chart {data-source="gui:doc/examples/charts/funnel-simple"}
+
+Here is an example of how to visualize a typical Sales process, looking at how
+many sale opportunities make it through, from the number of leads down to the
+number of actual sales.
+
+The data set is pretty simple:
+
+```py
+data = {
+    "Opps":  ["Hot leads", "Doc sent", "Quote", "Closed Won"],
+    "Visits": [316, 238, 125, 83]
+}
+```
+
+So from 316 leads, the company could retain 238 document exchanges, then 125 quotes.
+Finally, 83 opportunities were converted to actual close deals.
+
+In a Taipy funnel chart, the type of stage must be stored in the [*y*](../chart.md#p-y)
+property, and the volume in the [*x*](../chart.md#p-x) property of the chart control:
+
+!!! example "Page content"
+
+    === "Markdown"
+
+        ```
+        <|{data}|chart|type=funnel|x=Visits|y=Opps|>
+        ```
+  
+    === "HTML"
+
+        ```html
+        <taipy:chart type="funnel" x="Visits" y="Opps">{data}</taipy:chart>
+        ```
+
+The resulting chart looks like this:
+
+<figure>
+    <img src="../funnel-simple-d.png" class="visible-dark" />
+    <img src="../funnel-simple-l.png" class="visible-light"/>
+    <figcaption>Simple funnel chart</figcaption>
+</figure>
+
+
+### Multiple traces {data-source="gui:doc/examples/charts/funnel-multiple"}
+
+With the previous example, we could spot how many contacts would convert to sales,
+globally for a company.<br/>
+We can split this information into Regions and get more detailed conversion
+information.
+
+In the following code, we have the same information spread across regions:
+
+```py
+data = {
+    # The stage names
+    "Types": ["Website visit", "Downloads", "Prospects", "Invoice sent", "Closed"],
+    # Volume for each stage, in the US
+    "Visits_us": [13873, 10533, 5443, 2703, 908],
+    # Volume for each stage, in the EU
+    "Visits_eu": [ 7063, 4533, 3443, 1003, 1208 ],
+    # Volume for each stage, in the AP region
+    "Visits_ap": [ 6873, 2533, 3443, 1703, 508 ]
+}
+
+# Columns for each trace
+x = ["Visits_us", "Visits_eu", "Visits_ap"]
+
+# Legend text for each trace
+names = ["US", "EU", "AP"]
+```
+
+*x* is an array of the entries in the data set that we use as the
+value of the [*x*](../chart.md#p-x) property of the chart control.<br/>
+It is a shortcut to setting each [*x[]*](../chart.md#p-x) value, where we
+would have had to write, in the chart definition, the pretty long:
+`x[1]=Visits_us|x[2]=Visits_eu|x[3]=Visits_ap`, and even longer in HTML.
+
+The *names* variable holds all the trace names, so they appear properly in the chart
+legend.
+
+!!! example "Page content"
+
+    === "Markdown"
+
+        ```
+        <|{data}|type=funnel|chart|x={x}|y=Types|name={names}|>
```

Comment:
does this work ?
I would have thought {data}|chart|type=...

Reply:
Whao. Indeed this shouldn't work!
I messed up my copy/paste I guess. Thx!